app/helpers/application_helper/toolbar/x_vm_center.rb

```diff
@@ -310,7 +310,7 @@ class ApplicationHelper::Toolbar::XVmCenter < ApplicationHelper::Toolbar::Basic
           :confirm   => N_("The selected snapshot will be permanently deleted. Are you sure you want to delete the selected snapshot?"),
           :url_parms => "main_div",
           :onwhen    => "1",
-          :klass     => ApplicationHelper::Button::GenericFeatureButtonWithDisable,
+          :klass     => ApplicationHelper::Button::VmSnapshotRemoveOne,
           :options   => {:feature => :remove_snapshot}),
```

Comment:
@mzazrivec @borod108 the `:option` is in this case useless. The feature is hardcoded in the button code (https://github.com/ManageIQ/manageiq-ui-classic/pull/1628/files#diff-533cf50bb9cc5f712939b84bbbc9a0a1R7)